examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,14 @@
+''' An arc graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, start and
+end angles at a specified point.
```

Comment:
the line lengths can be made a bit more consistent
```suggestion
by drawing three arcs of defined radius, start and end angles
at a specified point.
```